core/src/it/scala/org/ensime/core/ScalapSymbolToFqnSpec.scala

```diff
@@ -31,16 +31,17 @@ class ScalapSymbolToFqnSpec extends EnsimeSpec
 
     val predef = vfs.vres("scala/Predef.class")
     val definedClassNames = new ClassfileDepickler(predef).getClasses
-    definedClassNames.foreach { scalaClass =>
-      verify(scalaClass.javaName, scalaClass.scalaName, scalaClass.declaredAs, cc)
+    definedClassNames.foreach {
+      case (_, scalaClass) =>
+        verify(scalaClass.javaName, scalaClass.scalaName, scalaClass.declaredAs, cc)
     }
   }
 
   it should "find and resolve field names in Predef" in withPresCompiler { (_, cc) =>
     val vfs = cc.vfs
 
     val predef = vfs.vres("scala/Predef.class")
-    val fieldNames = new ClassfileDepickler(predef).getClasses.flatMap(_.fields)
+    val fieldNames = new ClassfileDepickler(predef).getClasses.values.flatMap(_.fields.valuesIterator)
```

Comment:
`.values` is almost always better than `valuesIterator` as the latter exposes (mutable) implementation details